### PR TITLE
Fix JSHint errors

### PR DESCRIPTION
**Pull request by Shatakshi0805**
Signed-off-by: Shatakshi <shatakshi.gupta85@gmail.com>

**Description**
This PR fixes some JSHint errors as reported by [Sonatype Lift](https://lift.sonatype.com/result/meshery/meshery/01FHW0QAMP11ERCD7WXRTV9BKV?t=CustomTool%20%22JSHint%22%7CW004&t=CustomTool%20%22JSHint%22%7CW086&t=CustomTool%20%22JSHint%22%7CW069).

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/ConnectionWizard/index.js [hunk 1]
```diff
@@ -78,12 +78,16 @@ const ConfigurationWizard = () => {
     switch (step) {
       case 0:
         return <KubernetesScreen setStepStatus={setStepStatus}/>;
+        break;
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply ui/components/ConnectionWizard/index.js:81]: @Shatakshi0805 will you work to incorporate this feedback?

[the remark this answers is not in the document]
Comment (Shatakshi0805) [reply ui/components/ConnectionWizard/index.js:81]: yeah, on it. :+1:

**Comment by bariqhibat** [comment]
Any progress here? @Shatakshi0805

**Comment by Shatakshi0805** [comment]
> 

hi, i think i have made requested changes. if there's something else that needs to be done please let me know.

**Comment by bariqhibat** [comment]
> hi, i think i have made requested changes. if there's something else that needs to be done please let me know.

Yep, this is good enough. I think should be good to merge here! @leecalcote @navendu-pottekkat

**Review by bariqhibat — APPROVED** [review]

**Review by navendu-pottekkat — APPROVED** [review]

**Pull request merged**